app/assets/javascripts/features/confirm_dialog.js

```diff
@@ -0,0 +1,32 @@
+$(function(){
+  $(document).on('click', '.confirm-dialog-wrapper .close-dialog', function(ev){
+    $(this).closest('.confirm-dialog-wrapper').fadeOut(100);
```

Comment:
Could use a JS templates to build it each time, and then remove it after it's finished if you think that'd be a nicer way of handling it?